frontend/public/kubevirt/components/vm/vm-detail.jsx

```diff
@@ -30,31 +33,45 @@ import { Disk } from '../disk';
 import { NodeLink, LoadingInline } from '../../../components/utils';
 import { menuActions } from './menu-actions';
 
-const VmiEvents = ({obj: vm}) => {
-  const vmi = {
-    kind: VirtualMachineInstanceModel.kind,
-    metadata: {
-      name: vm.metadata.name,
-      namespace: vm.metadata.namespace,
-    },
+const VmEvents = ({ obj: vm }) => {
+  const vmObj = {
+    kind: vm.kind,
+    name: getName(vm),
+    namespace: getNamespace(vm),
+  };
+  const vmFilter = obj => _.isMatch(obj, vmObj);
```

Comment:
I would rather assign kind here for the consistency IMO

Reply:
kind added